spyder/app/mainwindow.py

```diff
@@ -2459,7 +2477,7 @@ def report_issue(self, body=None, title=None):
         if body is None:
             body = self.render_issue()
 
-        url = QUrl("https://github.com/spyder-ide/spyder/issues/new")
+        url = QUrl(__project_url__ + '/issues/new')

```

Comment:
This may be a little nitpicky, but since string concatenation isn't encouraged, maybe use format strings?

I'm getting spoiled by 3.6, `url = QUrl(f'{__project_url__}/issues/new')`, even though Spyder can't use that yet.

Reply:
> This may be a little nitpicky, but since string concatenation isn't encouraged, maybe use format strings?

Right, but isn't that just for concatenating more than two strings, and even then as a performance benefit on the order of microseconds unless you're concatenating a ton of strings or doing it in a loop? For two strings, both my own experience and what I've read has indicated a simple + is both clearest and just as fast.

> I'm getting spoiled by 3.6, `url = QUrl(f'{__project_url__}/issues/new')`, even though Spyder can't use that yet.

Same...

Reply:
No, you're right.